spec/helpers/application_helper_spec.rb

```diff
@@ -8,4 +8,11 @@
   it "returns enabled profiles" do
     expect(helper.disabled_profiles).to_not include('cataloging')
   end
+
+  it 'sets the title correctly' do
+    expect(helper.set_title('Mighty Poro', nil, nil)).to eq('Mighty Poro')
+    expect(helper.set_title('Mighty Poro', '', '')).to eq('Mighty Poro')
+    expect(helper.set_title('Mighty Poro', 'CollectionObect', '')).to eq('Mighty Poro (CollectionObect)')
+    expect(helper.set_title('Mighty Poro', 'Person', 'person')).to eq('Mighty Poro (Person) / (person)')
```

Comment:
So this would get changed to: 
    expect(helper.set_title('Mighty Poro', 'Person', 'person')).to eq('Mighty Poro (Person / person)')

Reply:
👍